src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,164 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static StorageDao storageDao;
+    private static RegistrationService registrationService;
+    private User mark;
+    private User sasha;
+    private User bob;
+    private User alice;
+    private User denis;
+
+    @BeforeAll
+    static void beforeAll() {
+        storageDao = new StorageDaoImpl();
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        mark = new User("Mark","123456", 18);
+        sasha = new User("Sasha","ValidPassword", 20);
+        bob = new User("Bob","short", 21);
+        denis = new User("Denis","isValid", 17);
+        alice = new User("Alice","PASSWORD", 19);
+    }
+
+    @Test
+    void setNullUser_NotOk() {
```

Comment:
You probably have to name your test methods starting with name of the method you're testing 
[Make sure you name your methods according to convention](https://google.github.io/styleguide/javaguide.html#s5.2.3-method-names)